examples/demo/cgi-bin/data.py

```diff
@@ -51,12 +51,23 @@ def exec_query(conn, title, query, response, keyspace=None, kr=None):  # pylint:
       cursor.begin()
       cursor.execute(query, {})
       cursor.commit()
+    # sanatize results (index columns are binary blobs)
+    sanatized_results = list(
```

Comment:
typo: `sanitized_results`.

However, instead of doing this, you can instead issue a `select ..., hex(keyspace_id) from name_idx` here: https://github.com/youtube/vitess/blob/master/examples/demo/cgi-bin/data.py#L162.

Reply:
oh yeah. This will be cleaner.